### ISSUE TITLE
figure out how to support FileAttachment

### ISSUE THREAD

**Issue by timelyportfolio** [juba/robservable#41]
`FileAttachment` does not seem to work.  Explore possibility of adding support for it.

**Comment by hrbrmstr** [comment]
ah, is this what is possibly making the following error?:

```
robservable(
  notebook = "@observablehq/plot",
  include = c(0:5)
)
```

![image](https://user-images.githubusercontent.com/509878/117148670-724efe80-ad84-11eb-82d2-6c91d3cb24f8.png)

**Comment by juba** [comment]
No, I think your error is linked to the fact that `Object.fromEntries` is not supported by the RStudio underlying web engine (a chromium version which is a bit outdated). It should work if you open it in a more recent browser, or if you run it in VSCode for example.

**Comment by hrbrmstr** [comment]
Except I'm on the dailies for pro and community edition :-)
![image](https://user-images.githubusercontent.com/509878/117162067-74b75580-ad90-11eb-859e-d7a0767e94a3.png)

**Comment by juba** [comment]
Yes, but I think the user agent string here (Chrome/69.0.3496.128) means that the underlying chromium version is the same as in stable. Did you try to open the result in a browser to see if it works ?

**Comment by hrbrmstr** [comment]
ah! (i'm daft) yep. i see what you mean now. it's fine in Edge/Chrome. 

and i think i got messed up b/c i hadn't tried it with and without a notebook that used `FileAttachment`.

apologies for messing up the issue. I'll delete the previous thread entries.

**Comment by timelyportfolio** [comment]
@hrbrmstr @juba maybe we should open another issue to build some plot examples and discuss some of these complicating issues.  Bob, not unexpected, we are thinking the same thing :)

**Comment by juba** [comment]
@timelyportfolio I wanted to take a look at this issue but I'm not sure what you mean precisely about the lack of support of `FileAttachment` ?
